tests/jeos/firstrun.pm

```diff
@@ -276,7 +276,7 @@ sub run {
         assert_screen 're-encrypt-finished', 600;
     }
 
-    unless (is_sle || is_sle_micro('<=6.0') || is_leap) {
+    if (is_tumbleweed || is_microos || is_sle_micro('>6.0') || is_sle('>=16')) {
```

Comment:
This is a big change in the programming logic, better to have a more conservative approach there

```suggestion
    unless (is_sle || is_sle_micro('<=6.0') || is_leap || is_sle('<16')) {
```

Reply:
Otherwise there will be side-effects because some things don't translate 1:1. E.g. `unless(is_leap)` is not the same as `if(is_tumbleweed)`. Many small different facets.

Reply:
I know, but if you look at the combinations that execute this code, the only places where this screen to create user is set is  TW, TW Microos, Micro 6.1+ and SLE16, the rest of the versions are left untouched

Reply:
Sure. If you're ok with having a look at the possible fallout, then I'm ok proceeding with the PR.